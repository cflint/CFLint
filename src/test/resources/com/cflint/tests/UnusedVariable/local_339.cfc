<cfcomponent>
	<cffunction name="foo">
		<cfset var local = {}>
		<cfset local.stEditor = 123>
		<cfscript>foo(local.stEditor)</cfscript>
	</cffunction>
</cfcomponent>
