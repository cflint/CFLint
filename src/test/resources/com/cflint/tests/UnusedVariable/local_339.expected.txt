{
  "version" : "",
  "timestamp" : 1526095034,
  "issues" : [ ],
  "counts" : {
    "totalFiles" : 0,
    "totalLines" : 0,
    "countByCode" : [ ],
    "countBySeverity" : [ ]
  }
}